src/components/SubCategory.js

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+import styled from 'styled-components';
+
+import Entry from './Entry';
+
+
+const StyledSubCategory = styled.div`
+  width: 80%;
+
+  .subCategory-header {
```

Comment:
**OPTIONAL** since JS variables are always camel case (first word lowercase, and subsequent uppercase), when naming classes sometimes it can look confusing, since `className="myClass"` and `className={myClass}` are two diff things (duh). For this reason, the convention i've seen for classnames is either to stick to all hyphens `my-long-class--header` or to capitalize it `MyLongClass--Header`. The goal is to visually distinguish a string value from a possible variable name.